libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/LocalizedString.kt

```diff
@@ -0,0 +1,45 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+/**
+ * A wrapper that contains a string in a particular language and the info
+ * about the corresponding language.
+ *
+ * @param language language which the string is in
+ * @param value original string value
+ */
+class LocalizedString internal constructor(
+    val language: String,
+    val value: String,
+) {
```

Comment:
What is the use case for this class, i.e. how are users supposed to use it and why we don't have the same structure for the other road objects or other stuff? Is it something applicable only for new route objects or we will be adopting it for other cases?

Reply:
The names for Junction/Interchange come as a map of language-value. This class represents the pair language-value. We used a similar approach for `EHorizonEdgeMetadata#names`, but there we have more than language and string so it's another class (`RoadComponent`).